tests/integration/test_writes/test_writes.py

```diff
@@ -262,6 +262,100 @@ def test_summaries(spark: SparkSession, session_catalog: Catalog, arrow_table_wi
     }
 
 
+@pytest.mark.integration
+def test_summaries_partial_overwrite(spark: SparkSession, session_catalog: Catalog) -> None:
+    identifier = "default.test_summaries_partial_overwrite"
```

Comment:
should we just add your jupyter notebook example to `dev/provision.py` and verify that the 2 operations produce the same snapshot summary?

Reply:
sounds good. Spark snapshot summary wont be changing anyways so its fine having just the comment.